pkg/flowkit/services/transactions.go

```diff
@@ -79,40 +79,133 @@ func (t *Transactions) GetStatus(
 	return tx, result, err
 }
 
-// Build builds a transaction with specified payer, proposer and authorizer.
-func (t *Transactions) Build(
+// NewTransactionAccounts defines transaction roles by accounts.
+//
+// You can read more about roles here: https://developers.flow.com/learn/concepts/accounts-and-keys
+func NewTransactionAccounts(
```

Comment:
minor: the naming isn't clicking with me...

How about something like: `NewTransactionAccountRoles` and `NewTransactionAccountRolesFromSingleAccount` (ok, that is a bit long), `transactionAccountRoles`, `transactionAddressRoles`...

Reply:
Yeah not great naming, couldn't think of others. NewTransactionAccountRoles this one I like.